docs/addons/argo-cd.md

```diff
@@ -120,6 +120,56 @@ blueprint.clone('us-west-2')
 
 The application promotion process in the above example is handled entirely through GitOps. Each stage specific App of Apps contains references to respective application GitOps repository for each stage (e.g referencing the release vs work branches or path-based within individual app GitOps repository).
 
+### GitOps for SSP AddOns
+
+By default all AddOns defined in SSP are deployed to the cluster via CDK. You can opt-in to deploy them following the GitOps model via ArgoCD. You will need a repository contains all the AddOns you would like to deploy via ArgoCD, such as, [ssp-eks-add-ons](https://github.com/aws-samples/ssp-eks-add-ons). You then configure ArgoCD bootstrapping with this repository as shown above.
```

Comment:
Let's drop the SSP from here and other places and replace in EKS Blueprints (or defined in a blueprint).

Reply:
Let us go  🚀